FreeRTOS/FreeRTOS-Kernel
# Style: Uncrustify kernel file - remove tab == 4 spaces

Style: Uncrustify kernel file

Description
-----------
Run uncrustify on kernel files
Insert indent on and off when uncrustify can't handle the format
Remove tabs from comments

Test Steps
-----------
No modification to source code


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
